src/components/forward/forward.js

```diff
@@ -66,14 +67,15 @@ class Forward extends Component {
     return (
       <div className={[style.controlButtonContainer, style.noIdleControl].join(' ')}>
         <Tooltip label={this.props.forwardText}>
-          <button
+          <Button
+            type={'button'}
```

Comment:
is needed? 
already got it in Button no?

Reply:
I don't think type button is needed for button tag